src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -3429,7 +3429,7 @@ public ShardingConfiguration ReadShardingConfiguration<TTransaction>(Transaction
         {
             using (var raw = ReadRawDatabaseRecord(context, name))
             {
-                return raw.Sharding.Value;
+                return raw.Sharding.MaterializedConfiguration;
```

Comment:
We need to ensure that this is something that can only be configured once (the prefixes setup, the actual distribution inside can change).